src/components/ui4/CardUi4.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
 
   // DO NOT USE after a scene is fully UI4! Margins should all align without adjustment.
   marginRem?: number[] | number
+  paddingRem?: number[] | number
```

Comment:
Suggestion: You can add a comment like `/* @deprecated Only here during the UI4 transition */` to each of these props, to get a strike-through effect whenever somebody uses them. This will make the restriction very obvious, right at the call sites.